hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -100,32 +103,45 @@ public Response get(
       @QueryParam("uploads") String uploads,
       @QueryParam("acl") String aclMarker,
       @Context HttpHeaders hh) throws OS3Exception, IOException {
-
-    if (aclMarker != null) {
-      S3BucketAcl result = getAcl(bucketName);
-      getMetrics().incGetAclSuccess();
-      return Response.ok(result, MediaType.APPLICATION_XML_TYPE).build();
-    }
-
-    if (uploads != null) {
-      return listMultipartUploads(bucketName, prefix);
-    }
-
-    if (prefix == null) {
-      prefix = "";
-    }
-
-
+    S3GAction s3GAction = S3GAction.GET_BUCKET;
+    Map<String, String> auditParams = S3Utils.genAuditParam(
```

Comment:
For audit purposes, it makes sense to capture the entire query sent and doing it centrally once similar to how the client IP is calculated. It will help identify bad clients who are not setting the correct params.

Reply:
Sure, I updated the generation of the audit params in a new ticket: https://github.com/apache/ozone/pull/3325/.

Could you help to check?